### ISSUE TITLE
[Component] [table-v2] Filter icon is shifting to a new line on hover

### ISSUE THREAD

**Issue by maxime-bc** [element-plus/element-plus#16114]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.4.21`
- Element Plus Version: `2.6.1`
- Browser / OS: `Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0`
- Build Tool: `Vite`

### Related Component

- `el-table-v2`

### Reproduction Link

[Github Repo](https://github.com/maxime-bc/test-tablev2-filter)

### Steps to reproduce

Hello, to demonstrate the issue I created a new project using `npm create vue@latest` with JSX support as it is needed by the `ElTableV2` component. Then, I just copied the example code snippet from the [ElTableV2 Filter](https://element-plus.org/en-US/component/table-v2.html#filter) section of the docs into the project main component, `App.vue`. 

### What is Expected?

When hovering over the header cell trying to click on the filter icon, it should not move.

### What is actually happening?

When hovering over the header cell trying to click on the filter icon, it shifts to the next line.

Please see the attached video:

![Kooha-2024-03-12-09-21-35](https://github.com/element-plus/element-plus/assets/49453941/e1b8beec-0acc-477a-8f5f-e82b83760b3c)

**Comment by zeyongTsai** [comment]
These CSS classes are not taking effect, perhaps you haven't installed the relevant atomic CSS library, such as UnoCSS/Tailwind, etc.

```jsx
<div class="flex items-center justify-center">
      <span class="mr-2 text-xs">{props.column.title}</span>
      <ElPopover ref={popoverRef} trigger="click" {...{ width: 200 }}>
```

**Comment by maxime-bc** [comment]
Hello, thanks for your help. Apparently, these CSS classes come from the Tailwind framework, which I did not install. I assumed these classes were from `element-plus`, as in the docs I can't find any mention of Tailwind being required for some examples. Maybe it is obvious that it is required, but I'm not yet comfortable with CSS frameworks. As I don't want to install a full framework for these small CSS classes, I just added this to the `style` block in my component:

```css
.flex {
  display: flex;
}

.items-center {
  align-items: center;
}

.justify-center {
  justify-content: center;
}

.mr-2 {
  margin-right: 0.5rem;
}

.text-xs {
  font-size: 0.75rem;
}

.cursor-pointer {
  cursor: pointer;
}
```
